test/codeu/chat/server/BasicControllerTest.java

```diff
@@ -38,7 +38,7 @@ public void doBefore() {
   @Test
   public void testAddUser() {
 
-    final User user = controller.newUser("user");
+    final User user = controller.newUser("user", "test");
```

Comment:
Should probably use something like "testPassword" so it's obvious what the parameter is.